lib/error-debug.js

```diff
@@ -11,11 +11,20 @@ export default ({ err: { name, message, stack, module } }) => (
     {
       name === 'ModuleBuildError'
       ? <pre style={styles.message} dangerouslySetInnerHTML={{ __html: ansiHTML(encodeHtml(message)) }} />
-      : <pre style={styles.message}>{stack}</pre>
+      : <StackTrace error={err} />
     }
   </div>
 )
 
+const StackTrace = ({ error: { name, message, stack } }) => (
+  <div>
+    <div style={styles.heading}>{name && message && `${name}: ${message}`}</div>
```

Comment:
So we may display `name` and `message` twice. Can you post the screenshot to see how it looks like ?